test/support/simple_websocket_client.ex

```diff
@@ -16,19 +16,23 @@ defmodule SimpleWebSocketClient do
     \r
     """)
 
-    expected = """
+    expected = ~r"""
     HTTP/1.1 101 Switching Protocols\r
+    date: [a-zA-Z]{3}, \d{1,2} [a-zA-Z]{3} \d{4} \d{2}:\d{2}:\d{2} GMT\r
     content-length: 0\r
     cache-control: max-age=0, private, must-revalidate\r
     upgrade: websocket\r
     connection: Upgrade\r
-    sec-websocket-accept: s3pPLMBiTxaQ9kYGzzhZRbK+xOo=\r
+    sec-websocket-accept: s3pPLMBiTxaQ9kYGzzhZRbK\+xOo=\r
     \r
     """
 
-    {:ok, response} = :gen_tcp.recv(client, byte_size(expected))
+    date_length = byte_size("date: #{Bandit.HTTP.date_header()}\r")
```

Comment:
This is a bit ugly, but since the day part of the date could be 1 or 2 digits the size of the expected response is not static.

Reply:
Shouldn't it always be the same length?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date

Reply:
> day part of the date could be 1 or 2 digits

I believe it should always be 2 digits, which means that it would always be the same length.

Reply:
It's wild that it took like 5 RFCs to get to this. HTTP/1.1 is so underspecified it's scary sometimes.